service/src/test/java/bio/terra/profile/pact/SamServiceTest.java

```diff
@@ -5,20 +5,19 @@
 import au.com.dius.pact.consumer.MockServer;
 import au.com.dius.pact.consumer.dsl.PactDslJsonBody;
 import au.com.dius.pact.consumer.dsl.PactDslWithProvider;
-import au.com.dius.pact.consumer.junit5.PactConsumerTestExt;
+import au.com.dius.pact.consumer.junit5.PactConsumerTest;
 import au.com.dius.pact.consumer.junit5.PactTestFor;
+import au.com.dius.pact.core.model.PactSpecVersion;
 import au.com.dius.pact.core.model.RequestResponsePact;
 import au.com.dius.pact.core.model.annotations.Pact;
 import bio.terra.profile.app.configuration.SamConfiguration;
-import bio.terra.profile.common.BaseUnitTest;
 import bio.terra.profile.service.iam.SamService;
 import org.junit.jupiter.api.Tag;
 import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.extension.ExtendWith;
 
 @Tag("pact-test")
-@ExtendWith(PactConsumerTestExt.class)
-public class SamServiceTest extends BaseUnitTest {
+@PactConsumerTest
+public class SamServiceTest {
```

Comment:
Because this extended `BaseUnitTest`, the test was being executed during the normal unit test phase (as well as during the pact test phase). With newer versions of pact, running it as a regular unit test failed.

The example pact tests do not extend anything, and I verified that the cases still run during the pact test phase.